### PR TITLE
Upload coverage results to CodeCov

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

**Comment by BjarkeMeier** [comment]
@djorgensendk, how about the required checks. They are not enabled and tests are failing in this PR.
Should we activate the required checks and thus blocking this PR or what do you think?

**Comment by djorgensendk** [comment]
@BjarkeMeier for sure, the required checks should be enabled, so that only tested stuff is committed to the main branch

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,56 +20,54 @@ on:
...
 
-  # databricks_ci:
-  #   uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@2.6.1
-  #   with:
-  #     PATH_STATIC_CHECKS: './source'
-  #     IGNORE_ERRORS_AND_WARNING_FLAKE8: 'E501,F401,E402,W503,E122,E123,E126,F811'
-  #     ARTIFACT_NAME: 'build-databricks_artifacts'
-  #     TEST_REPORT_PATH: ./source/databricks/htmlcov/
+  databricks_ci:
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@6.0.0
```
Comment (johevemi) [inline .github/workflows/ci.yml:51]:
```suggestion
    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@6.0.1
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -20,56 +20,54 @@ on:
...
 
-  # databricks_ci:
-  #   uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@2.6.1
-  #   with:
-  #     PATH_STATIC_CHECKS: './source'
-  #     IGNORE_ERRORS_AND_WARNING_FLAKE8: 'E501,F401,E402,W503,E122,E123,E126,F811'
-  #     ARTIFACT_NAME: 'build-databricks_artifacts'
-  #     TEST_REPORT_PATH: ./source/databricks/htmlcov/
+  databricks_ci:
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@6.0.1
```
Comment (johevemi) [inline .github/workflows/ci.yml:51]:
```suggestion
    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@6.1.1
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -20,56 +20,55 @@ on:
...
 
-  # databricks_ci:
-  #   uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@2.6.1
-  #   with:
-  #     PATH_STATIC_CHECKS: './source'
-  #     IGNORE_ERRORS_AND_WARNING_FLAKE8: 'E501,F401,E402,W503,E122,E123,E126,F811'
-  #     ARTIFACT_NAME: 'build-databricks_artifacts'
-  #     TEST_REPORT_PATH: ./source/databricks/htmlcov/
+  databricks_ci:
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@feature/test_docker
```
Comment (johevemi) [inline .github/workflows/ci.yml:52]:
```suggestion
    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@6.1.1
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -20,56 +20,55 @@ on:
...
 
-  # databricks_ci:
-  #   uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@2.6.1
-  #   with:
-  #     PATH_STATIC_CHECKS: './source'
-  #     IGNORE_ERRORS_AND_WARNING_FLAKE8: 'E501,F401,E402,W503,E122,E123,E126,F811'
-  #     ARTIFACT_NAME: 'build-databricks_artifacts'
-  #     TEST_REPORT_PATH: ./source/databricks/htmlcov/
+  databricks_ci:
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@6.1.1
```
Comment (johevemi) [inline .github/workflows/ci.yml:52]:
```suggestion
    uses: Energinet-DataHub/.github/.github/workflows/python-ci-test-and-coverage.yml@6.1.2
```

**Pull request merged**